scripts/main.py

```diff
@@ -308,6 +292,27 @@ def parse_arguments():
     except Exception as e:
         print_to_console("Error: \n", Fore.RED, str(e))
 
+    return command_name, arguments
+
+# TODO: Better argument parsing:
+# TODO: fill in llm values here
+
+cfg = Config()
+parse_arguments()
+ai_name = ""
+prompt = construct_prompt()
+# print(prompt)
+# Initialize variables
+full_message_history = []
+result = None
+# Make a constant:
+user_input = "Determine which next command to use, and respond using the format specified above:"
+
+# Interaction Loop
+while True:
+    # Send message to AI, get response
+    command_name, arguments = interaction_loop(cfg, prompt, mem, full_message_history)
+    
```

Comment:
Don't move code blocks around. If you think it's beneficial, submit this as a separate PR. Keep your diffs small.